src/networks.json

```diff
@@ -543,5 +543,15 @@
     "multicall": "0x537004440ffFE1D4AE9F009031Fc2b0385FCA9F1",
     "rpc": ["https://rpc.moonriver.moonbeam.network"],
     "explorer": "https://blockscout.moonriver.moonbeam.network/"
+  },
+    "336": {
+    "key": "336",
+    "name": "Shiden (Kusama)",
+    "shortName": "Shiden",
+    "chainId": 336,
+    "network": "mainnet",
+    "multicall": "0x8AfCFB1757489B89c73fd2E0bd8fF0a5D985a8d9",
+    "rpc": ["https://rpc.shiden.astar.network:8545"],
+    "explorer": "https://shiden.subscan.io"
```

Comment:
@m00nbeans @hskang9 I tested these again and i thiink `0x8AfCFB1757489B89c73fd2E0bd8fF0a5D985a8d9` is not a multicall strategy, 
I am getting the following error:
```
invalid type (argument="type", value="Call", code=INVALID_ARGUMENT, version=abi/5.4.1)
```
Make sure to pass a multicall contract here 
https://github.com/makerdao/multicall/blob/master/src/Multicall.sol
https://etherscan.io/address/0xeefba1e63905ef1d7acba5a8513c70307c1ce441#code

Reply:
multicall in shiden address is 0xeb8B87169549d524cA4C60165E550f520A3A48e0. @ChaituVR Please update this